ir-engine/ir-engine
# IR-4344 uploaded files extension tolower

## Summary
adding functionality to return a new File object with tolower on its extension, calls to this for upload files button and drag and drop files

## References
[https://tsu.atlassian.net/browse/IR-4344](https://tsu.atlassian.net/browse/IR-4344)

## QA Steps
Open dev env

Create a new scene

Upload mp3 file that has file extension of all caps ".MP3" (any supported file type works, just make it all caps as that was the root issue being fixed)

Observe that the uploaded file now displays with ".mp3" in lowercase (or whatever extension it should be)

Drag uploaded mp3 file from files tab to scene (again, any relevant file type, as long as it can be dragged into viewport - e.g. mp3, mp4...)

Observe the app behavior working as intended
